src/couch/test/couch_db_mpr_tests.erl

```diff
@@ -69,34 +70,36 @@ couch_db_mpr_test_() ->
                 foreach,
                 fun setup/0,
                 fun teardown/1,
-                [{with, [
+                [
```

Comment:
It's passed in from the foreach construct. It's similar to this this construct:

https://github.com/richcarl/eunit/blob/master/examples/eunit_examples.erl#L138-L140